### PR TITLE
Enable CI pylint check

### PR DESCRIPTION
**Pull request by sturivny**
Enable CI pylint check

### CODE REVIEW

**Comment by sturivny** [comment]
I've created two PRs:
https://github.com/oamg/convert2rhel/pull/36
https://github.com/oamg/convert2rhel/pull/35

I have no idea how to put project and action files in one commit.
Tried different variants but unsuccessfully.

**Comment by bocekm** [comment]
I've merged https://github.com/oamg/convert2rhel/pull/35, you can rebase this one.
I hope the .pylintrc in the root of the repo is the right place for the Pylint Runner to pick it up.

**Comment by sturivny** [comment]
Yeah, according to the documentation: 
```
Pylint searches for a configuration file in the following order and uses the first one it finds:

pylintrc in the current working directory
.pylintrc in the current working directory
```

**Comment by sturivny** [comment]
Looks like it works without rebasing, now we got:
```
Your code has been rated at 8.57/10
```

**Comment by bocekm** [comment]
Great, and now it's the time to fix the code based on the pylint output where it makes sense :)

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  pylint_check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 2.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 2.7
```
Comment (bocekm) [inline .github/workflows/pylint.yml:18]: Is it possible to run the checks on both 2.6 and 2.7? Because RHEL 6 has py2.6.

Comment (sturivny) [reply .github/workflows/pylint.yml:18]: I'll check it out

**Comment by sturivny** [comment]
@bocekm pylint for python2.6 is enabled and run into CentOS 6 docker container, as Ubuntu doesn't support python2.6 anymore.
So now we have:
- pylint for python2.6 in CentOS 6 docker container
- pylint for python2.7 in CentOS 7 docker container

Please check the commit and if everything is ok you can merge it.

**Pull request merged**